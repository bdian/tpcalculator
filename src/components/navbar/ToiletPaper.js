import React from 'react';
import styled from 'styled-components';
import { useSpring, animated, config } from 'react-spring';

const Toiletpaper = (props) => {
  const ripAnimation = useSpring({
    from: { transform: 'translate3d(0, 0, 0)' },
    transform: 'translate3d(0, -20em, 0)',
    config: config.wobbly
  });

  return (
    <>
      <ToiletPaper style={ripAnimation}>
      </ToiletPaper>
    </>
  );
};

export default Toiletpaper;

const ToiletPaper = styled.div`
    position: fixed;
    width: 100%;
    top: -10px;
    left: 0;
    background: #2d3436;
    z-index: 1;
    font-size: 1.4rem;
    border-top: 2px dotted white;
    border-bottom: 2px dotted white;
    height: 600px;
  `;
