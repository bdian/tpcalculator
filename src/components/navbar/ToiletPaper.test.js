import React from 'react';
import ToiletPaper from './ToiletPaper';
import renderer from 'react-test-renderer';

test('Toilet Paper snapshot', () => {
  Date.now = jest.fn(() => 1482363367071);
  const component = renderer.create(
    <ToiletPaper />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
