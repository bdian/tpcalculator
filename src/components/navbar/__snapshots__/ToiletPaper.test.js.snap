// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toilet Paper snapshot 1`] = `
<div
  className="sc-AxjAm fcsFOA"
  style={
    Object {
      "transform": AnimatedInterpolation {
        "attach": [Function],
        "calc": [Function],
        "children": Array [],
        "detach": [Function],
        "payload": Array [
          AnimatedValue {
            "animatedStyles": Set {},
            "children": Array [],
            "done": false,
            "lastPosition": 0,
            "lastTime": undefined,
            "lastVelocity": undefined,
            "payload": undefined,
            "setValue": [Function],
            "startPosition": 0,
            "startTime": 1482363367071,
            "value": 0,
          },
        ],
      },
    }
  }
/>
`;
